content/en/docs/troubleshooting/ResourcePlacement.md

```diff
@@ -0,0 +1,206 @@
+
+In this case we have `ResourcePlacement` called test-rp in namespace `test-ns`.
+
+1. List the `ResourcePlacement` to get the name of the RP,
+
+```bash
+kubectl get rp test-rp -n test-ns
+NAME       GEN   SCHEDULED   SCHEDULEDGEN   AVAILABLE   AVAILABLE-GEN   AGE
+test-rp    1     True        1              True        1               15s
+```
+
+2. The following command is run to view the status of the `ResourcePlacement` deployment.
+
+```bash
+kubectl describe resourceplacement test-rp -n test-ns
+```
+
+3. Here's an example output. From the `Placement Statuses` section of the `test-rp` status, notice that it has distributed
+resources to two member clusters and, therefore, has two `ResourceBindings` instances:
+
+```bash
+Status:
+  Conditions:
+  - Last Transition Time: "2023-11-23T00:49:29Z"
+    ...
+  Placement Statuses:
+  - Cluster Name: kind-cluster-1
+    conditions:
+      ...
+      Reason: AllWorkHaveBeenApplied
+      Status: True
+      Type: Applied
+  - clusterName: kind-cluster-2
+    conditions:
+      ...
+      Reason: AllWorkHaveBeenApplied
+      Status: True
+      Type: Applied
+```
+
+3. To get the `ResourceBindings` value, run the following command:
+
+```bash
+    kubectl get resourcebinding -n test-ns -l kubernetes-fleet.io/parent-CRP=test-rp 
+```
+
+4. The output lists all `ResourceBindings` instances that are associated with `test-rp`.
+
+```bash
+kubectl get resourcebinding -n test-ns -l kubernetes-fleet.io/parent-CRP=test-rp 
+NAME                              WORKSYNCHRONIZED   RESOURCESAPPLIED   AGE
+test-rp-kind-cluster-1-be990c3e   True               True               33s
+test-rp-kind-cluster-2-ec4d953c   True               True               33s
+```
+
+The `ResourceBinding` resource name uses the following format: `{RPName}-{clusterName}-{suffix}`.
+Find the `ResourceBinding` for the target cluster you are looking for based on the `clusterName`.
+
+## How can I find the latest ResourceSnapshot resource?
+
+To find the latest ResourceSnapshot resource, run the following command:
+
+```bash
+kubectl get resourcesnapshot -n <namespace> -l kubernetes-fleet.io/is-latest-snapshot=true,kubernetes-fleet.io/parent-CRP={RPName}
+```
+
+> NOTE: In this command, replace `{RPName}` with your `ResourcePlacement` name and `<namespace>` with the namespace where the ResourcePlacement exists.
+
+## How can I find the correct work resource that's associated with ResourcePlacement?
+
+To find the correct work resource, follow these steps:
+
+1. Identify the member cluster namespace and the `ResourcePlacement` name. The format for the namespace is `fleet-member-{clusterName}`.
+2. To get the work resource, run the following command:
+
+```bash
+kubectl get work -n fleet-member-{clusterName} -l kubernetes-fleet.io/parent-CRP={RPName}
+```
+
+> NOTE: In this command, replace `{clusterName}` and `{RPName}` with the names that you identified in the first step.
+
+## Important Considerations for ResourcePlacement
```

Comment:
this probably should be moved ahead of the rest as this might be the first thing user wants to know